projects/plugins/jetpack/_inc/client/product-descriptions/index.jsx

```diff
@@ -44,7 +44,7 @@ const ProductDescriptions = props => {
 
 	return (
 		<>
-			<QueryProducts />
```

Comment:
Do you know if there's any information queried in the products call that's not queried by the site products call?

Reply:
Before this change, the product sent to the ProductDescription component was :
```json
{
	"title": "Jetpack Site Search",
	"slug": "jetpack_search",
	"key": "search",
	"description": "Help your site visitors find answers instantly so they keep reading and buying. Great for sites with a lot of content.",
	"features": [
		"Instant search and indexing",
		"Powerful filtering",
		"Supports 29 languages",
		"Spelling correction"
	],
	"available": true,
	"currencyCode": "USD",
	"showPromotion": true,
	"promotionPercentage": 50,
	"includedInPlans": [],
	"fullPrice": 59.95,
	"upgradeUrl": "https://jetpack.com/redirect/?source=jetpack-product-description-checkout&path=jetpack_search&site=munson1996.jurassic.tube"
}
```

After, it is:
```json
{
	"title": "Jetpack Site Search",
	"slug": "jetpack_search",
	"key": "search",
	"description": "Help your site visitors find answers instantly so they keep reading and buying. Great for sites with a lot of content.",
	"features": [
		"Instant search and indexing",
		"Powerful filtering",
		"Supports 29 languages",
		"Spelling correction"
	],
	"available": true,
	"currencyCode": "USD",
	"showPromotion": true,
	"promotionPercentage": 50,
	"includedInPlans": [],
	"fullPrice": 59.95,
	"saleCoupon": {
		"start_date": "2022-06-20 00:00:00",
		"expires": "2023-06-20 00:00:00",
		"discount": 50,
		"purchase_types": [
			3
		],
		"product_ids": [
			2104
		],
		"allowed_for_domain_transfers": false,
		"allowed_for_renewals": false,
		"allowed_for_new_purchases": true,
		"code": "b07ac1e15cb15678"
	},
	"upgradeUrl": "https://jetpack.com/redirect/?source=jetpack-product-description-checkout&path=jetpack_search&site=munson1996.jurassic.tube"
}
```

It appears to have all of the values as before plus a few extra 😄